CoreUtilities/src/au/gov/asd/tac/constellation/utilities/text/StringUtilities.java

```diff
@@ -412,6 +413,6 @@ public static String removeSpecialCharacters(final String originalString) {
             }
             return expected;
         }
-        return new ArrayList<>();
+        return Collections.EMPTY_LIST;
```

Comment:
Use `Collections.emptyList()` rather than `Collections.EMPTY_LIST` (the former generates a typed list whereas the latter generates a raw list)